README.md

```diff
@@ -59,6 +59,42 @@ If you have AWS credits on your account and want to see them taken into account
         --param "credits_remaining=xxx.xx"
     ```
 
+## Support for other Dimensions
+
+If you have and AWS Organisation established and would like to see a breakdown by account, you can override the default dimensions with parameters: 
+
+    ```
+    serverless deploy \
+        --param "slack_url=https://hooks.slack.com/services/xxx/yyy/zzzz" \
+        --param "group=LINKED_ACCOUNT" \
+        --param "group_length=15"
+    ```
+
+Possible value for `group` are:
```

Comment:
Can you include a link to the AWS docs where these groups come from?

Reply:
I actually got them from the error message when I specified an invalid value.

Reply:
From - https://docs.aws.amazon.com/cli/latest/reference/ce/get-cost-and-usage.html#options

> Valid values for the DIMENSION type are AZ , INSTANCE_TYPE , LEGAL_ENTITY_NAME , INVOICING_ENTITY , LINKED_ACCOUNT , OPERATION , PLATFORM , PURCHASE_TYPE , SERVICE , TENANCY , RECORD_TYPE , and USAGE_TYPE .

Reply:
Nice, thanks!